packages/opencensus-instrumentation-grpc/test/test-grpc.ts

```diff
@@ -508,7 +508,7 @@ describe('GrpcPlugin() ', function() {
               grpcModule.status.OK
             );
             if (method.method === grpcClient.unaryMethod) {
-              assertStats(testExporter, 233);
+              assertStats(testExporter, 2582);
```

Comment:
See comment in https://github.com/census-instrumentation/opencensus-node/pull/928/files#r675003588

probably because of bumped `object-sizeof` dependency (or gRPC encoding differences).